builtin/logical/aws/path_config_rotate_root.go

```diff
@@ -39,44 +40,44 @@ func (b *backend) pathConfigRotateRootUpdate(ctx context.Context, req *logical.R
 		return nil, err
 	}
 	if rawRootConfig == nil {
-		return logical.ErrorResponse("no configuration found for config/root"), nil
+		return nil, fmt.Errorf("No configuration found for config/root")
```

Comment:
Sorry, I forgot to mention the other part. For `logical.ErrorResponse` we start with upper case words, but for server errors we start with lower case words.

Reply:
My fault, I knew that but forgot. Fixed.